src/components/ScrollableTabView/ScrollableTabView.js

```diff
@@ -104,12 +92,17 @@ class ScrollableTabView extends React.Component {
     this.children().map((child, idx) => {
       const key = this.makeSceneKey(child, idx)
       const { currentPage, containerWidth, sceneKeys } = this.state
+      const {minViewportHeight } = this.props
```

Comment:
I think that linter somehow missed this space before minViewportHeight

Reply:
fixed